tests/fdb/test_fdb_mac_expire.py

```diff
@@ -0,0 +1,258 @@
+    FDB_INFO_FILE = "/root/fdb_info.txt"
+    POLLING_INTERVAL_SEC = 15
+    FDB_AGING_TIMER_EXPIRE_WINDOW = 15
+
+    def __getFdbTableCount(self, duthost, mac):
+        """
+            Gets number of FDB table entries containing mac entry.
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+                mac (str): MAC value to search for in the FDB table
+
+            Returns:
+                (int) representing the number of the FDB entries containing th eMAC address
+        """
+        result = duthost.shell("show mac | grep {0} | wc -l".format(mac))
+        pytest_assert(result["rc"] == 0, "Failed to obtain mac address with error '{0}'".format(result["stderr"]))
+
+        return int(result["stdout"])
+
+    def __loadSwssConfig(self, duthost):
+        """
+            Load SWSS configuration on DUT
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Raises:
+                asserts if the load SWSS config failed
+
+            Returns:
+                None
+        """
+        result = duthost.shell(argv=[
+            "docker",
+            "exec",
+            "swss",
+            "bash",
+            "-c",
+            "swssconfig /etc/swss/config.d/switch.json"
+        ])
+        pytest_assert(result["rc"] == 0, "Failed to load swss config with error '{0}'".format(result["stderr"]))
+
+    def __deleteTmpSwitchConfig(self, duthost):
+        """
+            Delete temporary switch.json cofiguration files
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        result = duthost.find(path=["/tmp"], patterns=["switch.json*"])
+        for file in result["files"]:
+            duthost.file(path=file["path"], state="absent")
+
+    def __runPtfTest(self, ptfhost, testCase='', testParams={}):
+        """
+            Runs FDB MAC Expire test case on PTF host
+
+            Args:
+                ptfhost (AnsibleHost): Packet Test Framework (PTF)
+                testCase (str): FDB tests test case name
+                testParams (dict): Map of test params required by testCase
+
+            Returns:
+                None
+
+            Raises:
+                RunAnsibleModuleFail if ptf test fails
+        """
+        pytest_assert(ptfhost.shell(argv=[
+            "ptf",
+            "--test-dir",
+            "ptftests",
+            testCase,
+            "--platform-dir",
+            "ptftests",
+            "--platform",
+            "remote",
+            "-t",
+            ";".join(["{0}={1}".format(k, repr(v)) for k, v in testParams.items()]),
+            "--relax",
+            "--debug",
+            "info",
+            "--log-file",
+            "/tmp/{0}".format(testCase)
+            ],
+            chdir = "/root",
+            )["rc"] == 0,
+            "Failed when running test '{0}'".format(testCase)
+        )
```

Comment:
Similar issue. If rc of ptfhost.shell is not 0, a exception will be raised. There is no chance to check `rc` value here.